src/components/breadcrumb/SingleBreadcrumb.tsx

```diff
@@ -37,7 +38,16 @@ export function SingleBreadcrumb({ first = false, flexWidth = 1, onClick = null,
       <div className="position-absolute chevronDiv top-50 start-50 translate-middle d-flex">
         {workbench ? (
           workbench.index === ordino.focusViewIndex ? (
-            <FilterAndSelected />
+            <>
+              <FilterAndSelected />
+              <button
+                type="button"
+                className="btn btn-icon-light btn-sm align-middle m-1"
+                onClick={() => dispatch(setCommentsOpen({ workbenchIndex: workbench.index, open: !workbench.commentsOpen }))}
+              >
+                <i className="flex-grow-1 fas fa-comments" />
+              </button>
```

Comment:
@dg-datavisyn Do we want to enable/disable comments in Ordino via config flag? If yes, is that a global flag or do we want to change it also for selected entities?

Reply:
In general the comments functionality should only be available when comments is part of the bundle. Disable/Enable comments on an entity basis is out of scope. The commercial feature set and its technical integration needs to be discussed on a more general level first. Since this will block this PR, we decided to keep the integration for as it is for now and address it in a later issue.